doc/source/templates/01_batch_inference/start.ipynb

```diff
@@ -8,17 +8,20 @@
    "source": [
     "# Scaling Batch Inference with Ray Data\n",
     "\n",
-    "This template is a quickstart to using [Ray Data](https://docs.ray.io/en/latest/data/dataset.html) for batch inference. Ray Data is one of many libraries under the [Ray AI Runtime](https://docs.ray.io/en/latest/ray-air/getting-started.html). See [this blog post](https://www.anyscale.com/blog/model-batch-inference-in-ray-actors-actorpool-and-datasets) for more information on why and how you should perform batch inference with Ray!\n",
+    "| Template Specification | Description |\n",
+    "| ---------------------- | ----------- |\n",
+    "| Summary | This template walks through GPU batch inference on an image dataset using a PyTorch ResNet model. |\n",
+    "| Time to Run | Less than 2 minutes to compute predictions on the dataset. |\n",
+    "| Minimum Compute Requirements | No hard requirements. The default is 4 nodes, each with 1 NVIDIA T4 GPU. |\n",
+    "| Cluster Environment | This template uses the latest Anyscale-provided Ray ML image using Python 3.9: [`anyscale/ray-ml:2.4.0-py39-gpu`](https://docs.anyscale.com/reference/base-images/ray-240/py39#ray-ml-2-4-0-py39). If you want to change to a different cluster environment, make sure that it is based off of this image! |\n",
     "\n",
-    "This template walks through GPU batch prediction on an image dataset using a PyTorch model, but the framework and data format are there just to help you build your own application!\n",
+    "By the end, we will have classified > 3000 images using the pre-trained ResNet model and saved these predictions to a local directory.\n",
     "\n",
-    "At a high level, this template will:\n",
```

Comment:
Not pointing people at the material in the Ray Data docs is a little unfortunate, especially since with Ray 2.5 we will have a lot of great guides there.

We should figure out better what the relation between the workspace template and the material in the docs is

cc @zhe-thoughts @praveen-palanisamy

Reply:
These links got moved to an appendix in the notebook itself. This change was due to some feedback from the bugbash. The template was giving too much information to the user up front.